src/admin/erldns_admin.erl

```diff
@@ -7,25 +7,38 @@ This application will read from your `sys.config` the following example:
 ```erlang
 {erldns, [
     {admin, [
-        {credentials, {<<"username">>, <<"password">>}},
         {port, 8083},
-        {middleware, [my_custom_middleware, another_middleware]}
+        {tls, false},
+        {credentials, {~"username", ~"password"}},
```

Comment:
QQ, why are me moving these binaries to atoms?

Reply:
Not atoms, but erlang's improved notation for binary strings using sigils, I've changed that in many other places but the API was a barely reworked and modernised part of the codebase. The idea is that ~"" implies a utf8 encoded binary string, but other sigils can be used like ~r"" for compiled regular expressions and don't remember what else :)